### ISSUE TITLE
Revert not working

### ISSUE THREAD

**Issue by codyhatch** [martyjs/marty-devtools#11]
When I click “revert to action” the tool pane updates to show the old state, but my app doesn’t rerender, and investigating my actual app state isn’t changing.

As an example, my app fires a `RECEIVE_LOCATIONS` action when a bunch of location data is loaded and stored in my locationStore.  In the dev tools, I can see the action, and click "revert to action" on a prior action.  `RECEIVE_LOCATIONS` vanishes, and the locationStore visible in the tool empties.

However, my app is still rendering the locations, and the actual locationStore is still full of locations.  And if any other action is triggered, the tool will "snap" back to showing the full locationStore.

In short, the dev tool can't seem to actually change the state of my app; it just pretends.  I'm using v0.10 of marty (but saw the same behaviour in v0.9) and the latest dev tools.

**Comment by Shearerbeard** [comment]
I can confirm that I'm seeing the same behavior - Revert to action updates the stores panel on the right properly but does not trigger a re-render of the app. Upon firing a new action in the app the state updates as if the previous revert never took place. Marty 0.10 and 0.90. Chrome 43 on OS X 10.10.3
